app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -137,3 +137,113 @@ def test_create_new_experiment_remote_settings_timeout(
             break
     else:
         raise AssertionError("Timeout text was never shown.")
+
+
+def test_end_experiment_and_approve_end(
+    selenium, base_url, default_data, create_experiment, request, perform_kinto_action
+):
+    default_data.public_name = request.node.name
+
+    selenium.get(base_url)
+    home = HomePage(selenium, base_url).wait_for_page_to_load()
+    current_experiments = None
+    try:
+        current_experiments = len(home.tables[0].experiments)
+    except TimeoutException:
+        current_experiments = 0
+    review = create_experiment(selenium, home, default_data)
+    review.launch_without_preview.click()
+    review.request_review.click_launch_checkboxes()
+    review.request_review.request_launch_button.click()
+    review.approve()
+
+    perform_kinto_action(selenium, base_url, "approve")
+
+    selenium.get(base_url)
+    # refresh until the experiment shows up
+    for attempt in range(45):
```

Comment:
We bumped the number of retries up to 60 and that seemed to help with intermittency so we should probably set these to 60 too.  Or maybe pull it out into a constant at the top?

Reply:
lets do a constant _fixture_ 😉

Reply:
We should use the timeout from the .env file. Let me make that an enhancement.

Reply:
If you're talking about the KINTO_TIMEOUT we'd want double that here.  But we should make this 60 before we land it and then do it as a followup?

Reply:
Wait, I'm confused. Do we mean this timeout?

https://github.com/mozilla/experimenter/blob/d723f2978f2dd420a661cdd8642925ec62535125/.env.integration-tests#L29

If I'm reading the code correctly, it feels a bit odd to have it serve as both a maximum timeout value (in seconds) and a maximum loop counter value.

Reply:
Yeah that's why I said double it, because it won't happen in exactly that time, it'll happen in some multiple of the kinto sync time, so doubling it to gives the test enough time for the kinto sync to happen and the frontend to pick it up.